docs/api/registration-base-url-resource.md

```diff
@@ -237,6 +237,16 @@ Other        | The package is deprecated due to a reason not on this list
 
 If the `reasons` property contains strings that are not from the known set, they should be ignored. The strings are case-insensitive, so `legacy` should be treated the same as `Legacy`. There is no ordering restriction on the array, so the strings can arranged in any arbitrary order. Additionally, if the property contains only strings that are not from the known set, it should be treated as if it only contained the "Other" string.
 
+#### Alternate package
+
+The alternate package object has the following properties:
+
+Name         | Type   | Required | Notes
+------------ | ------ | -------- | -----
+id           | string | yes      | The ID of the alternate package
+range        | object | no       | The allowed [version range](../concepts/package-versioning.md#version-ranges-and-wildcards), or `*` if any version is allowed
```

Comment:
`or * is latest version is recommended` -- thoughts @scottbommarito @karann-msft?